lib/author.rb

```diff
@@ -0,0 +1,21 @@
+class Author
+  attr_reader :books
+
+  def initialize(info)
+    @first_name = info[:first_name]
+    @last_name = info[:last_name]
+    @books = []
+  end
+
+  def name
+    "#{@first_name} #{@last_name}"
```

Comment:
This is great - You could also make an ivar for `@name` and do this manipulation in the `initialize` method.